templates/monthly-update.md

```diff
@@ -130,13 +128,6 @@ things. Give us a shout-out and we will link it from this section! ✍
 
 #### Flux Adopters shout-out
 
-<!-- 
-
-Please consult https://github.com/fluxcd/website/commits/main/data/adopters
-for this.
-
--->
-
```

Comment:
I think it's good to leave a comment here so folks have a reminder where to look for this kind of information.